Blockchain/readme.md

```diff
@@ -11,10 +11,14 @@
 - [Smart Contracts](https://ethereum.org/en/developers/docs/smart-contracts/)
 - [Crypto wallet](https://www.businessinsider.com/personal-finance/crypto-wallet)
 - [Block Explorers](https://www.gemini.com/cryptopedia/what-is-a-block-explorer-btc-bch-eth-ltc)
+- [Blockchain Layer 1 vs.Layer 2 Scaling Solutions](https://www.gemini.com/cryptopedia/blockchain-layer-2-network-layer-1-network)
 - [Oracles](https://chain.link/education/blockchain-oracles)
 - [What Are NFTs](https://www.forbes.com/advisor/investing/cryptocurrency/nft-non-fungible-token/)
 - [What iS A DAO](https://www.forbes.com/sites/cathyhackl/2021/06/01/what-are-daos-and-why-you-should-pay-attention/?sh=343b04067305)
 - [Intro To DAPPS](https://ethereum.org/en/developers/docs/dapps/)
+- [EVM and Non-EVM Chains](https://ethereum.org/en/developers/docs/dapps/) ```
+
+A syntax error which is causing the markdown to render incorrectly. 
```

Comment:
Unwanted additional lines. Please fix it

Reply:
Can I know how can I make changes or do I need to open a new PR?

Reply:
You can simply add commits in your branch and they will be reflected in the PR

Reply:
DONE ! Removed the unwanted lines .